packages/solidstart/test/server/withServerActionInstrumentation.test.ts

```diff
@@ -10,9 +10,9 @@ import {
   spanToJSON,
 } from '@sentry/node';
 import { NodeClient } from '@sentry/node';
-import { solidRouterBrowserTracingIntegration } from '@sentry/solidstart/solidrouter';
 import { redirect } from '@solidjs/router';
 import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
+import { solidRouterBrowserTracingIntegration } from '../../src/client/solidrouter';
```

Comment:
l: Odd change to have in this pr 😁 Any reason?

Reply:
Notice it's wrong and depends on the package already being built. I can extract this into its own PR if you prefer.

Reply:
No all good :)